server/core/src/main/java/dev/slimevr/bridge/ProtobufBridge.java

```diff
@@ -198,7 +198,7 @@ protected void trackerAddedReceived(TrackerAdded trackerAdded) {
 			remoteTrackersBySerial.put(tracker.getName(), tracker);
 		}
 		synchronized (remoteTrackersByTrackerId) {
-			remoteTrackersByTrackerId.put(tracker.getId(), tracker);
+			remoteTrackersByTrackerId.put(tracker.getTrackerNum(), tracker);
```

Comment:
I think this is generally supposed to be persistent, but I don't think we are entirely able to do that right now. Why not use ID though?

Reply:
Because that uses the SteamVR tracker ID, which I changed to be the trackerNum 
![image](https://github.com/SlimeVR/SlimeVR-Server/assets/20461725/f90f0213-e727-45b1-9bbb-f0949fa49c3b)